frontend/src/pages/HomePage.react.tsx

```diff
@@ -145,7 +150,13 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
     WpApi
       .get('display-welcome-panel')
       .end((err: any, res: request.Response) => {
-        if (res.body === true) {
+        if (err) {
+          return;
+        }
+        const body = Array.isArray(res.body)
+          ? res.body[0]
+          : res.body;
```

Comment:
Why is this ternary condition on three adjacent lines ? Is it intentional or not.